app/views/order_numbers/_order_number.html.erb

```diff
@@ -1,63 +1,55 @@
-          </tr>
-          <tr class="govuk-table__row">
-            <th scope="col" class='govuk-table__header'>Suspension period start</th>
-            <td class="numerical govuk-table__cell numerical"><%= order_number.definition.suspension_period_start_date.to_formatted_s(:full) %></td>
-          </tr>
-          <tr class="govuk-table__row">
-            <th scope="col" class='govuk-table__header'>Suspension period end</th>
-            <td class="numerical govuk-table__cell numerical"><%= order_number.definition.suspension_period_end_date.to_formatted_s(:full) %></td>
-          </tr>
-          <tr class="govuk-table__row">
-            <th scope="col" class='govuk-table__header'>Blocking period start</th>
-            <td class="numerical govuk-table__cell numerical"><%= order_number.definition.blocking_period_start_date.to_formatted_s(:full) %></td>
+            <th scope="col" class='govuk-table__header'>Last allocation date</th>
+            <td class="numerical govuk-table__cell numerical"><%= order_number.definition.last_allocation_date.to_formatted_s(:gov) || 'n/a' %></td>
           </tr>
           <tr class="govuk-table__row">
-            <th scope="col" class='govuk-table__header'>Blocking period end</th>
-            <td class="numerical govuk-table__cell numerical"><%= order_number.definition.blocking_period_end_date.to_formatted_s(:full) %></td>
+            <th scope="col" class='govuk-table__header'>Suspension period</th>
+            <td class="numerical govuk-table__cell numerical"><%= suspension_period_dates_for(order_number.definition) %></td>
           </tr>
           <tr class="govuk-table__row">
-            <th scope="col" class='govuk-table__header'>Last allocation date</th>
-            <td class="numerical govuk-table__cell numerical"><%= order_number.definition.last_allocation_date.to_formatted_s(:full) %></td>
+            <th scope="col" class='govuk-table__header'>Blocking period</th>
+            <td class="numerical govuk-table__cell numerical"><%= blocking_period_dates_for(order_number.definition) %></td>
           </tr>
         </tbody>
       </table>
-      <p class="govuk-!-margin-top-4">In an effort to facilitate UK importers HMRC endeavours to keep quotas 'OPEN' where possible, therefore the information displayed above may differ slightly from the current UK position. If you require further information contact the Tariff Quota Management Team by email: <%= mail_to("tariff.management@hmrc.gsi.gov.uk", nil, subject: "Order number #{order_number.number}") %>.</p>
+      <p class="govuk-!-margin-top-4">In an effort to facilitate UK importers HMRC endeavours to keep quotas 'OPEN' where possible, therefore the information displayed above may differ slightly from the current UK position. If you require further information contact the Tariff Quota Management Team by email: <%= mail_to("tariff.managment@hmrc.gov.uk", nil, subject: "Order number #{order_number.number}") %>.</p>
```

Comment:
The `managment` part of this address seems misspelled.
I've seen it's also misspelled in the ticket, is the official address that is misspelled or it's a typo from Matt?

Reply:
I clarified with Matt and this should have the e in it. This is the spelling in the ticket

Reply:
```suggestion
      <p class="govuk-!-margin-top-4">In an effort to facilitate UK importers HMRC endeavours to keep quotas 'OPEN' where possible, therefore the information displayed above may differ slightly from the current UK position. If you require further information contact the Tariff Quota Management Team by email: <%= mail_to("tariff.management@hmrc.gov.uk", nil, subject: "Order number #{order_number.number}") %>.</p>
```